app/views/service/_svcs_show.html.haml

```diff
@@ -3,6 +3,10 @@
     %ul.nav.nav-tabs{'role' => 'tablist'}
       = miq_tab_header("details") do
         = _("Details")
+      - if @record.type == "ServiceTerraformTemplate"   
+        - stack = @record.try(:stack, "Provision")
```

Comment:
This is extremely close to `- provision_job = @record.try(:job, "Provision")` below, I wonder if we could drop the "Is this an AnsiblePlaybook service or is it a terraform service" and instead say "does the provision job support stdout" generally

Reply:
Example https://github.com/ManageIQ/manageiq-ui-classic/pull/9292